index.js

```diff
@@ -197,7 +199,37 @@ function init (ssb, config) {
       const data = keystore.group.get(id)
       if (!data) return cb(new Error(`unknown groupId ${id})`))
 
-      cb(null, data)
+      // find if this group has any parent links
+      scuttle.link.findGroupBySubgroupId(id, (err, links) => {
```

Comment:
does this method find groups or links.... the name suggests it would return groups, but your variable there is "links"
can we rename it for clarity / consistency please

Reply:
My bad, it finds groups

Reply:
Updated :)